crates/runtime/Cargo.toml

```diff
@@ -118,6 +118,7 @@ snafu.workspace = true
 snowflake-api = { workspace = true, optional = true }
 spicepod = { path = "../spicepod" }
 ssh2 = { workspace = true, optional = true }
+strsim = "0.11.1"
```

Comment:
This should be gated by the `models` feature

Reply:
Good idea.